src/dual.jl

```diff
@@ -279,6 +279,12 @@ Base.trunc(d::Dual) = trunc(value(d))
 Base.round(::Type{R}, d::Dual) where {R<:Real} = round(R, value(d))
 Base.round(d::Dual) = round(value(d))
 
+if VERSION ≥ v"1.4"
+    Base.div(x::Dual, y::Dual, r::RoundingMode) = div(value(x), value(y), r)
+else
+    Base.div(x::Dual, y::Dual) = div(value(x), value(y))
+end
```

Comment:
Do we also need overloads for `div` when one of the arguments is not a Dual?
e.g. for if literals are involved?

Reply:
What is a literal?

Reply:
e.g. if the code is `div(10, x)`

Reply:
Ah okay, yes that is implicitly supported via promotion. I just added an explicit test.